geopandas/base.py

```diff
@@ -859,6 +859,39 @@ def interiors(self):
         """
         return _delegate_property("interiors", self)
 
+    def remove_repeated_points(self, tolerance=0.0):
+        """Returns a ``GeoSeries`` containing a copy of the input geometry
+        with repeated points removed.
+
```

Comment:
```suggestion

        Parameters
        ----------
        tolerance : float
            radius within which a point is considered to be a repeat of another


```
Not sure how to phrase this, but think it's worth clarifying what tolerance actually is.
This is also missing from the shapely documentation, but empirically it's the radius/ distance and accounts for both coordinates:
```
shapely.remove_repeated_points(Polygon([(0, 0), (.35, .35), (0, 1), (.5, 1), (0,0)]), tolerance=.5)
Out[38]: <POLYGON ((0 0, 0 1, 0 0, 0 0))>
shapely.remove_repeated_points(Polygon([(0, 0), (.36, .36), (0, 1), (.5, 1), (0,0)]), tolerance=.5)
Out[39]: <POLYGON ((0 0, 0.36 0.36, 0 1, 0 0))>
```